str_analysis/data/known_pathogenic_variant_catalog__GRCh37__58_loci_without_offtargets.json

```diff
@@ -0,0 +1,1405 @@
+[
```

Comment:
basic question, why do you need two json files? couldn't you store all of the loci without off target regions and then store a file just containing the off target regions for the loci that have them?